crates/noirc_evaluator/src/ssa.rs

```diff
@@ -49,6 +49,7 @@ pub(crate) fn optimize_into_acir(
         ssa = ssa
             .inline_functions()
             .print(print_ssa_passes, "After Inlining:")
+            .evaluate_assert_constant()?
```

Comment:
No extra `.print` after this call since I figured it'd clutter output too much for too little of a change